scripts/preprocessing/03-make_epochs.py

```diff
@@ -88,14 +110,23 @@ def run_epochs(subject, session=None):
 
     # Epoch the data
     msg = (f'Creating epochs with duration: '
-           f'[{config.epochs_tmin}, {config.epochs_tmin}] sec')
+           f'[{config.epochs_tmin}, {config.epochs_tmax}] sec')
     logger.info(gen_log_message(message=msg, step=3, subject=subject,
                                 session=session))
+
+    reject = config.get_rejects()
+    if reject == 'auto':
+        reject = _get_global_reject_epochs(
+                raw, tmin=config.epochs_tmin,
+                tmax=config.epochs_tmax,
+                events=events,
+                event_id=event_id)
+
```

Comment:
I've changed the indentation slightly
```suggestion
        reject = _get_global_reject_epochs(
            raw, tmin=config.epochs_tmin,
            tmax=config.epochs_tmax,
            events=events,
            event_id=event_id
        )
```